tools-v2/pkg/cli/command/curvebs/status/client/client.go

```diff
@@ -123,6 +123,10 @@ func (cCmd *ClientCommand) Print(cmd *cobra.Command, args []string) error {
 }
 
 func (cCmd *ClientCommand) RunCommand(cmd *cobra.Command, args []string) error {
+	if len(cCmd.metrics) == 0 {
```

Comment:
when users use this command directly(he don't create any client), they will get nothings by this way. This is strange for users.

Reply:
IMO, nothing output means no client in cluster is acceptable.